app/services/katello/pulp3/content.rb

```diff
@@ -29,7 +29,7 @@ def upload_chunk(upload_href, offset, content, size)
             filechunk.write(content)
             filechunk.flush
             actual_chunk_size = File.size(filechunk)
-            uploads_api.update(upload_href, content_range(offset, offset + actual_chunk_size - 1, size), filechunk)
+            uploads_api.update(content_range(offset, offset + actual_chunk_size - 1, size), upload_href, filechunk)
```

Comment:
opened an issue here to write a vcr test: https://projects.theforeman.org/issues/30777